test/ecto/changeset_test.exs

```diff
@@ -799,6 +799,13 @@ defmodule Ecto.ChangesetTest do
     changeset = changeset(%{"decimal" => Decimal.new(1.5)})
                 |> validate_number(:decimal, greater_than_or_equal_to: Decimal.new(1.5))
     assert changeset.valid?
+
+    changeset = changeset(%{"decimal" => Decimal.new(-1)})
+                |> validate_number(:decimal, greater_than_or_equal_to: -1.5)
+    assert changeset.valid?
+    changeset = changeset(%{"decimal" => Decimal.new(1.5)})
+                |> validate_number(:decimal, greater_than_or_equal_to: 1.5)
+    assert changeset.valid?
```

Comment:
Will add one test with an integer and one with float instead two tests with float.